platform/podjs/README.md

```diff
@@ -20,21 +20,48 @@ feature. While we could support the creation of a container and a
 mechanism for loading features in `pod.js` without too much trouble,
 it is currently not implemented.
 
-The `polyOut` and `polyNav` interfaces are not implemented at the
-moment. The former because we don't use it yet, the latter because it
-is currently not part of the `Pod` interface, which we aim to rectify
-soon.
-
 ## Building
 
-Simply run `build.js` in the repository root.
+Simply run `./build.js` in the repository root.
+
+## How to set up a feature to build with podjs
+
+Follow these instructions to do so:
+
+1.  You need to point to `podjs` from the `polypoly-eu` repository and
+    add it as a "development dependency" on your project by running:
+
+    `$npm i <path-that-points-to-platform/podjs --D`
+
+    So, you end up having on your `package.json` the following:
+
+    ```json
+       "devDependencies": {
+          "@polypoly-eu/podjs": "file: <path-that-points-to-platform/podjs>",
+          ...
+       },
+
+    ```
+
+2.  The main step is to copy `pod.js` in your project. For this, please, check the `example` feature as guideline. You should be able to do it with `rollup` tooling (same way as in `example`) or any other script mechanism you desire.
+
+3.  Make sure your manifest data feature file is exposed as `window.manifestData`.
```

Comment:
Tripping up on "manifest data feature file is exposed" -> There is no "manifest data feature file".

There's a "feature manifest", or a "feature manifest file", which is `manifest.json`. The contents of that need to be exposed as the `manifestData` property on the `window` object in order for podjs to be able to read it.

So to make a constructive suggestion, I guess I'd say: "Make sure your `manifest.json` is exposed as `window.manifestData`."